public/js/signupdom.js

```diff
@@ -0,0 +1,73 @@
+const form = document.getElementById("registerForm");
+var username = document.getElementById("username");
+var email = document.getElementById("email");
+var password = document.getElementById("password");
+var massage = document.getElementById("massage");
+var checkbox = document.getElementById("chkOwner");
+var shopname = document.getElementById("shopname");
+var address = document.getElementById("address");
+  
+form.addEventListener("submit", event => {
+  event.preventDefault();
+  if (!checkbox.checked) {
+    validateUsernameEmailPassword();
```

Comment:
it would be better if you use the joi validation in the backend rather than this validation :smiley:

Reply:
humm , ok when we set together will change :1234: